### PR TITLE
Add payment flow to table registration

### PR DESCRIPTION
**Pull request by Silbeg**
Some other tweaks as well for look and feel for events, and table registration.

### CODE REVIEW

**Comment by Silbeg** [comment]
Committing to master, so that I can test things at https://test-pfs-registration.herokuapp.com

**Comment by Silbeg** [comment]
Please make comment in either case.

File: app/controllers/registration_tables_controller.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@ class RegistrationTablesController < ApplicationController
   before_action :get_event, :get_session, :get_table, :get_registration_tables, :get_possible_players
 
 
+  PREMIUM_MESSAGE = "This is a premium table. If payment isn't received within 1 hour, we will have to remove you from this table."
```
Comment (panpear) [inline app/controllers/registration_tables_controller.rb]: I'd reword this part to something like: "This is a premium table, requiring payment. If payment is not received within an hour, you will be removed from this table". Or remove this message, as it is redundant with the message provided in the registration_tables html.

File: app/controllers/registration_tables_controller.rb [hunk 2]
```diff
@@ -84,25 +86,42 @@ def create
     @registration_table = RegistrationTable.new(registration_table_params)
     respond_to do |format|
       if @registration_table.save
-        format.html { redirect_to [@event], notice: 'Table was successfully added.' }
-        format.json { render :show, status: :created, location: [@event] }
+        if @registration_table.payment_ok?
+          format.html {redirect_to [@event], notice: 'Table was successfully added.'}
+        else
+          format.html {redirect_to new_event_session_table_registration_table_table_payment_path(@event, @session, @table, @registration_table), notice: PREMIUM_MESSAGE}
```
Comment (panpear) [inline app/controllers/registration_tables_controller.rb]: Is this standard ruby notation for redirections? (I'm just assuming it is, there are similar ones elsewhere). It gets a little hard to read with the word table in there three times (esp. the ...registration_table_table_payment...). Not sure if there's a good way to address this though.

Comment (Silbeg) [inline app/controllers/registration_tables_controller.rb]: Oh, trust me, its "fun". This is railsy stuff, which I am probably not doing the best of. There is a way to collapse the length of that, but yeah, this is using the routes "properly"
